spec/services/ccms/requestors/opponent_organisation_search_requestor_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+module CCMS
+  module Requestors
+    RSpec.describe OpponentOrganisationSearchRequestor do
+      let(:expected_xml) { ccms_data_from_file "get_common_org_request.xml" }
+      let(:expected_tx_id) { "20230904121530000000" }
+      let(:organisation_name) { "Bucks Council" }
+      let(:organisation_type) { "LA" }
+      let(:requestor) { described_class.new("my_login", organisation_name, organisation_type) }
+
+      describe "XML request" do
```

Comment:
You could add a test to check the header contains the correct XML payload. But this is probably missing from the Apply specs too?!

Reply:
👍 have added a test to check the header contains correct payload